app/helpers/patients_helper.rb

```diff
@@ -73,4 +73,11 @@ def dcaf_pledge_limit_help_text
 
     "Pledge Limit Guidelines:<br />1st trimester (7-12 weeks): $#{first_tri}<br />2nd trimester (12-24 weeks): $#{second_tri}<br />Later care (25+ weeks): $#{later_care}"
   end
+
+  def duration_since_last_call
+    @patient = Patient.find(params[:id])
+    days_since_last_call = (Time.now - @patient.calls.last.created_at).to_i / 86400
```

Comment:
Saying it again -- I really love how elegant this is. I would have written something way more complicated, but this nails it and is way nicer.

I think there are a few ways we can tighten this up a little bit -- interested in your thoughts:
* mongo tends to be spotty with maintaining orders of stuff. Can we do a `sort_by` on created at so we can be sure to pick the right last one?
* can we assign the 86400 to a variable (eg `DAY = 86400`), so we can be absolutely sure what the number refers to? This is burned into my brain too, but we can't assume that for everyone necessarily.

Have you written a test before? If so (or if you're interested in learning!) then this would be a great thing to write a test for in `test/helpers/status_helper_test.rb`, but if not, that's cool too!